components/docker-up/docker-up/runc

```diff
@@ -0,0 +1,2 @@
+This file is downloaded as part of the build.
```

Comment:
is this due to new go workspace leeway setup?

Reply:
It's mainly for convenience. Without those files present, we'd see an (inconsequential) error when opening a workspace.